prow/report/report.go

```diff
@@ -233,10 +235,14 @@ func createComment(reportTemplate *template.Template, pj kube.ProwJob, entries [
 		"--- | --- | --- | ---",
 	}
 	lines = append(lines, entries...)
+	if reportTemplate != nil {
+		lines = append(lines, []string{
+			"",
+			b.String(),
+			"",
+		}...)
```

Comment:
There is no need to use a slice literal here, just pass the elements as args to append.